src/components/Button.js

```diff
@@ -0,0 +1,41 @@
+/*
+		This is the new button component that I made as a result of adding Radium.  Here I can keep my code 'DRY'
+		by having just one Button component.  I wanted to remove CSS files completely by using Radium for this app
+		and found it to be impossible to not repeat code unless the app is truly reusable with its components.
+ */
```

Comment:
This is the new button component that I made as a result of adding Radium.  Here I can keep my code 'DRY' by having just one Button component.  I wanted to remove CSS files completely by using Radium for this app and found it to be impossible to not repeat code unless the app is truly reusable with its components.
So now you can pass `buttonContent` a string as props and set the button text, `onClick` sets the method when you click the button.